source/camera/DummyArrayClass.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ *  Copyright (C) 2011-2015  OpenDungeons Team
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "camera/DummyArrayClass.h"
+
+#include <algorithm>
+
+void DummyArrayClass::sort()
+{
+    std::vector<Vector3i> ll[4];
```

Comment:
That would indeed be a nice name ;-)
You know, usually, if you don't know how to name a variable, that's because there is a flaw in your code. And usually, this kind of variable ends called "myArray" and such.